Makefile

```diff
@@ -10,10 +10,16 @@ test: build-normal
 build: build-normal build-optimized
 
 build-normal:
-	cargo build --target wasm32-unknown-unknown --release
+	cargo build --target wasm32-unknown-unknown --release -p soroban-token-contract
+	cp target/wasm32-unknown-unknown/release/soroban_token_contract.wasm ./
+	cargo build --target wasm32-unknown-unknown --release -p soroban-liquidity-pool-contract
+	cp target/wasm32-unknown-unknown/release/soroban_liquidity_pool_contract.wasm ./
+	cargo build --target wasm32-unknown-unknown --release -p soroban-single-offer-contract
+	cp target/wasm32-unknown-unknown/release/soroban_single_offer_contract.wasm ./
```

Comment:
@sisuresh I added in `cp` commands to copy these files out on every build so we don't get into an issue where they are stale. I actually think you could change to just referencing the `target/` directories. I do that in some other examples and it works fine as long as the crate have a dependency in such a way they build in the right order, or they are named alphabetically to order that way. I didn't want to get into the weeds on that though, so I didn't make that change.